storage/aptosdb/src/lru_node_cache.rs

```diff
@@ -9,6 +9,7 @@ use lru::LruCache;
 
 const NUM_SHARDS: usize = 256;
 
+#[derive(Debug)]
```

Comment:
This will print a lot in the log if used.. shall we implement it manually?

Reply:
Done